src/components/Profile/components/EmergencyInfoList/index.js

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+import './index.css';
+import ProfileInfoListItem from '../ProfileInfoListItem';
+import user from 'services/user';
+
+import { Typography, Grid, Card, CardHeader, CardContent, List } from '@material-ui/core';
+import { useEffect } from 'react';
+
+const formatPhone = (phone) => {
+  let tele = String(phone);
+  if (tele.length === 10) {
+    return '(' + tele.slice(0, 3) + ') ' + tele.slice(3, 6) + '-' + tele.slice(6);
+  } else if (tele !== 'undefined') {
+    return tele;
+  }
+};
+
+const EmergencyInfoList = ({ username }) => {
+  const [emrg, setEmrg] = useState([]);
```

Comment:
I think you could come up with a clearer name for this variable. Maybe `contacts`?
```suggestion
  const [contacts, setContacts] = useState([]);
```

Or, since long variable names are usually better than unclear variable names, you could make it `emergencyContacts`.